web/src/app/lists/PaginatedList.tsx

```diff
@@ -82,7 +81,7 @@ function PageControls(props: {
       alignItems='center'
       className={classes.controls}
     >
-      <Grid item>
+      <CardActions>
```

Comment:
I think we want to keep the controls outside of the card, unless that was a design decision.

current impl:

<img width="621" alt="Screen Shot 2022-02-04 at 12 07 23 PM" src="https://user-images.githubusercontent.com/11381794/152596200-9e84bc13-4a28-4f3c-902d-e535f6b6390e.png">

incoming change:

<img width="679" alt="Screen Shot 2022-02-04 at 12 06 57 PM" src="https://user-images.githubusercontent.com/11381794/152596127-09a8bc50-518e-4673-ab1c-03894800ad89.png">

Reply:
Right, I forgot that's why the PageControls were pulled out of the PaginatedList file. I switched it back and cleaned up the context so it's just using state now.